windows build error blocking 0.18.0 release

https://www.travis-ci.com/github/zaquestion/lab/builds/212062231
```
  ⨯ release failed after 257.68s error=failed to build for windows_386: go: downloading github.com/inconshreveable/mousetrap v1.0.0

go: downloading github.com/Microsoft/go-winio v0.4.15

# github.com/zaquestion/lab/cmd

cmd/util.go:333:25: cannot use syscall.Stdout (type syscall.Handle) as type int in argument to terminal.IsTerminal

cmd/util.go:333:65: cannot use syscall.Stderr (type syscall.Handle) as type int in argument to terminal.IsTerminal
```

After we fix this, just need to recreate the `v0.18.0` release and travis should roll it out. cc @bmeneguele @prarit

--- a/cmd/util.go
+++ b/cmd/util.go
@@ -330,7 +330,8 @@ func textToMarkdown(text string) string {
 // to avoid some markdown rendering garbage going to other outputs that
 // don't support some control chars.
 func isOutputTerminal() bool {
-	if !terminal.IsTerminal(syscall.Stdout) || !terminal.IsTerminal(syscall.Stderr) {
+	if !terminal.IsTerminal(int(syscall.Stdout)) ||
+		!terminal.IsTerminal(int(syscall.Stderr)) {
 		return false
 	}
 	return true

